SecurityPrivacyQuestionnaire.md

```diff
@@ -0,0 +1,184 @@
+distinct information that may not be obvious to users.
+
+Examples of how data could be exposed that may not be obvious to users:
+
+* If the Controlled Frame is used similar to &lt;iframe&gt;, all concerns
+about how an &lt;iframe&gt; may obfuscate the origin source apply to Controlled
+Frame. The IWA will need to take care to ensure that users understand the source
+of the embedded content before interacting with it.
+
+* If users interact with embedded content within a Controlled Frame, the
+embedder could exfiltrate that interaction via JavaScript.
+
+* An embedded could interact with embedded content and use APIs that carry
+some associated exposure risk.
+
+## [2.6 Do the features in your specification introduce state that persists across browsing sessions?](https://www.w3.org/TR/security-privacy-questionnaire/#persistent-origin-specific-state)
+
+Controlled Frame is only available to isolated contexts (TODO: link).
+
+The methods in the specification currently do not store state that perists
+across browsing sessions. As an embedding control, though, Controlled Frame
+allows state to be stored across browsing sessions for any origin that is loaded
+within it.
+
+This is a key reason why Controlled Frame is only available to isolated
+contexts.  Isolated contexts store their data in a separate storage partition
+from the default storage partition used in normal web browsing. Any data stored
+in a way that persists will be stored in this separate storage partition.
+
+Controlled Frame by default will not persist its data unless the `partition`
+attribute contains the "persist:" prefix. Otherwise, all of the data in the
+embedding control instance will be stored in memory for the user agent to
+discard once the control no longer needs it.
+
+## [2.7 Do the features in your specification expose information about the underlying platform to origins?](https://www.w3.org/TR/security-privacy-questionnaire/#underlying-platform-data)
+
+By default, third-party origins loaded within a Controlled Frame are not aware
+that they are embedded within another page. To the page embedded within a
+Controlled Frame, the page appears to be the top-level page and has no exposed
+parent the way that, say, an &lt;iframe&gt; would. This allows the page to
+continue to function while being embedded and provides safety to the embedding
+page from the embedded page.
+
+While an embedder has a Controlled Frame instance, it's possible that through
+interacting with the instance using its methods or events, information about the
+underlying platform could be exposed. This exposure could be either purposefully
+or accidentally due to, for example, the types of functionality available, such
+as JavaScript execution.
+
+## [2.8 Does this specification allow an origin to send data to the underlying platform?](https://www.w3.org/TR/security-privacy-questionnaire/#send-to-platform)
+
+Controlled Frame does not support overriding existing operating system APIs
```

Comment:
Do you mean that it doesn't provide a means of communicating with the underlying platform?

Reply:
Yes, I updated the wording here.